.github/workflows/pull-request-checks.yml

```diff
@@ -203,6 +203,26 @@ jobs:
       - name: Cargo audit
         run: cargo audit
 
+  # Check: licenses (only use approved list of licenses), unmaintained repos, yanked versions
+  cargo-deny:
+    name: Run cargo-deny
```

Comment:
This sounds as unrelated to this PR but to that one: https://github.com/thin-edge/thin-edge.io/pull/3451/files

Reply:
oops. yeah that slipped in.

Reply:
fixed.